libs/designsystem/src/lib/components/chart-wip/chart-js/chart-js.service.spec.ts

```diff
@@ -0,0 +1,865 @@
+import { ElementRef } from '@angular/core';
+import { Chart, FontSpec } from 'chart.js';
+import { AnnotationOptions } from 'chartjs-plugin-annotation';
+
+import { ColorHelper } from '../../../helpers';
+import { CHART_ANNOTATION_CONFIGS, CHART_TYPE_CONFIGS } from '../chart-wip.configs';
+import { CHART_GLOBAL_DEFAULTS } from '../chart-wip.configs';
+import { ChartDataset } from '../chart-wip.types';
+import { ChartHighlightedElements } from '../chart-wip.types';
+
+import { ChartJSService } from './chart-js.service';
+
+describe('ChartJSService', () => {
+  let chartJSService: ChartJSService;
+  let canvasElement: ElementRef<HTMLCanvasElement>;
+
+  beforeEach(() => {
+    chartJSService = new ChartJSService();
+    const nativeElement = document.createElement('canvas');
+    canvasElement = new ElementRef<HTMLCanvasElement>(nativeElement);
+  });
+
+  describe('function: renderChart', () => {
+    describe('when annotations are given', () => {
+      it('should render the chart with annotations applied', () => {
+        const annotations: AnnotationOptions[] = [
+          { type: 'line', yMin: 10, yMax: 10 },
+          { type: 'line', yMin: 20, yMax: 20 },
+        ];
+
+        chartJSService.renderChart({
+          targetElement: canvasElement,
+          type: 'bar',
+          data: [1, 2, 3],
+          dataLabels: ['one', 'two', 'three'],
```

Comment:
This part of the object is reused quite a lot, consider extracting to a variable. 
```suggestion
          ...options,
```

Reply:
I get your points - but in order to keep the tests [isolated & flat](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test#the-good-test-is-isolated-and-flat) I think the correct way about it is to duplicate the options (eventough it is very WET)

Reply:
Good point, I was probably still a bit in component code review mode when I wrote this 😄